zuul.d/jobs.yaml

```diff
@@ -1094,13 +1094,15 @@
     parent: ansible-test-network-integration-iosxr
     abstract: true
     vars:
+      ansible_test_collections: true
```

Comment:
this is not correct, again we still test on ansible/ansible so that is not a collection